download.jl

```diff
@@ -0,0 +1,82 @@
+#Extracting wind data from MET file 
+download("https://earthsciml.s3.amazonaws.com/compressed_Input/met/mcipv5/METDOT3D_160701.bson")
+
+#Change the name after downloading the file from AWS. 
+#The directories will vary by users 
+
+file = "/var/folders/s3/d0ks9vxn08dclbqqy0q3tgzw0000gn/T/jl_MRWMZx"
+mv(file, "METDOT3D_160701.bson")
+
+#Use decompress_netcdf and the change .bson file to .nc
+#The function is defined below
+#The directories will vary by users 
+comp_file = "/Users/Minwoo/Desktop/METDOT3D_160701.bson"
+output = "/Users/Minwoo/Desktop/METDOT3D_160701.nc"
```

Comment:
If the decompressed file name is inferred from the name of the compressed file as suggested above, then this is not needed.